src/armature/Import.ts

```diff
@@ -26,30 +26,51 @@ class Data {
         this.lines = rawData.split('\n');
     }
 
-    public readLine() {
-        let line = this.lines[0].split(' ');
-        this.lines.shift();
-        // Skip comments and empty lines.
-        while (line.length < 1 || line[0].startsWith('#')) {
-            line = this.lines[0].split(' ');
-            this.lines.shift();
-        }
-
-        return line;
-    }
-
-    // TODO: Refactor to handle each line type individually and multipblex
-    // (so we're not reliant on a particular order)
-    public getLinesWithPrefix(prefix: string) {
-        const lines: string[][] = [];
-        while (this.lines[0].startsWith(prefix)) {
-            lines.push(this.readLine());
-        }
-
-        return lines;
+    public read() {
```

Comment:
`read` is kinda general, maybe `parse` or something like that would be a more descriptive name?